ang/crmCaseType.js

```diff
@@ -354,9 +371,11 @@
 
       // go lookup and add client-perspective labels for $scope.caseType.definition.caseRoles
       _.each($scope.caseType.definition.caseRoles, function (set) {
-        _.each($scope.relationshipTypeOptions, function (relTypes) {
-          if (relTypes.text == set.name) {
+        _.each($scope.relationshipTypeOptionsAll, function (relTypes) {
+          if (relTypes.xmlName == set.name) {
             set.displaylabel = relTypes.id;
```

Comment:
I think this is a spot where the original comparison was done backwards, but because it was just among labels, it didn't matter too much.  Now, it could be an issue.  Comparing the `xmlName` to the name stored in the XML is correct, but the displayLabel should be the `text`--the client's perspective, as used in the UI:

```suggestion
             set.displaylabel = relTypes.text;
```